packages/firestore/test/integration/api/composite_index_query.test.ts

```diff
@@ -136,4 +149,157 @@ apiDescribe('Composite Index Queries', persistence => {
       });
     });
   });
+
+  describe('Aggregation queries - sum / average', () => {
+    it('aggregate query supports collection groups - multi-aggregate', () => {
+      const testHelper = new CompositeIndexTestHelper();
+      return testHelper.withTestCollection(persistence, async (coll, db) => {
+        const collectionGroupId = coll.id;
+        const docPaths = [
+          `${collectionGroupId}/cg-doc1`,
+          `abc/123/${collectionGroupId}/cg-doc2`,
+          `zzz${collectionGroupId}/cg-doc3`,
+          `abc/123/zzz${collectionGroupId}/cg-doc4`,
+          `abc/123/zzz/${collectionGroupId}`
+        ];
+        const batch = writeBatch(db);
```

Comment:
Does writeBatch work with the test helper library you created for composite index queries? If not, perhaps we should migrate this code over to use the addDoc(...) method that will add TTL fields and more?

Reply:
I used ` testHelper.addTestSpecificFieldsToDoc` to add the TTL and testID fields, with minimal change to the existing code:
```
batch.set(
    doc(db, docPath), 
    testHelper.addTestSpecificFieldsToDoc({ a: 2 }) // original: { a: 2 }
);
```
If there is no preference between write Batch and addDoc(), I can refactor the code. @MarkDuckworth